common/src/main/java/org/opensearch/ml/common/transport/agent/MLExecuteAgentAction.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.ml.common.transport.agent;
+
+import org.opensearch.action.ActionType;
+
+/**
+ * ML execute agent action.
+ */
+public class MLExecuteAgentAction extends ActionType<MLExecuteAgentResponse> {
```

Comment:
Need to register the action in [getActions](https://github.com/opensearch-project/ml-commons/blob/main/plugin/src/main/java/org/opensearch/ml/plugin/MachineLearningPlugin.java#L465) extension point

Reply:
added in the new commit https://github.com/opensearch-project/ml-commons/pull/4007/commits/6cfd04ebc650c6840b327d3f2e325a8e05c615a5